### PR TITLE
Calculate system energy.

### PR DESCRIPTION
**Pull request by marjanAlbouye**
This PR implements methods needed to calculate system energy based on center-to-center distances. Default energy function calculates energy based on inverse of distance squared. 
User can specify energy function when creating an instance of the Simulation class.

### CODE REVIEW

File: MCMC/simulation.py [hunk 1]
```diff
@@ -1,9 +1,25 @@
...
 import math
 
+import numpy as np
+
+from utils import inverse_distance_energy
+
+
 class Simulation:
-    def __init__(self, n_density=0.5, r=0.1, L=5, kT=1.0, r_cut=1, max_trans=0.2, write_freq=5):
+    def __init__(self, n_density=0.5, r=0.1, L=5, kT=1.0, r_cut=1, max_trans=0.2, write_freq=5,
```
Comment (chrisjonesBSU) [inline MCMC/simulation.py:10]: IMO, it is most simple to have a default radius of 1, then L is the count of radii. So, in the case of default r=1.0 and L=50; the box lengths are 25 circles long.  Technically this is no different than what you are doing it, but let's us start with easier numbers to work with.

File: MCMC/simulation.py [hunk 2]
```diff
@@ -17,6 +33,7 @@ def __init__(self, n_density=0.5, r=0.1, L=5, kT=1.0, r_cut=1, max_trans=0.2, wr
         self.accepted_moves = 0
         self.tps = None
         self.write_freq = write_freq
+        self.energy_func = energy_func
```
Comment (chrisjonesBSU) [inline MCMC/simulation.py:37]: It doesn't look like an attribute is being set for maximum allowed transition?

**Review by chrisjonesBSU — COMMENTED** [review]
A couple comments, but looks good to me!

File: MCMC/simulation.py (hunk 2 above)
Comment (marjanAlbouye) [reply MCMC/simulation.py:37]: A few lines above this line.

**Pull request merged**